workflow/wdl/workflow.wdl

```diff
@@ -1,139 +1,64 @@
 version 1.0
 
-# import "tasks/nextstrain.wdl" as nextstrain  # <= modular method
+# import "tasks/buildfile.wdl" as buildfile
+import "tasks/nextstrain.wdl" as nextstrain  # <= modular method
+# import "tasks/ncov_ingest.wdl" as ncov_ingest
 
 workflow Nextstrain_WRKFLW {
   input {
-    # Option 1: run the ncov example workflow
-    File? build_yaml
-
-    # Option 2: create a build_yaml from sequence and metadata
+    # Option 1: Pass in a sequence and metadata files, create a build_yaml
     File? sequence_fasta
     File? metadata_tsv
     String? build_name
-    # It's possible all of the above files are provided
+
+    # Option 2: Use a custom build_yaml file with https or s3 sequence or metadata files
+    File? build_yaml
```

Comment:
Oh! Yes, easy to rename now or later. Just need to change two wdl files, and probably update the rst

Reply:
Renaming would break people's existing workflow parameter sets (in Terra or otherwise) though, right?

Reply:
Well, depends on how people are using workflow parameters... Erik and I are using data.tables (would break), but we'd just have to update the workflow card (pretty easy). 

@huddlej seems to prefer using the file selector, so not much change for him.

Reply:
Changed `build_yaml` to `configfile_yaml`, reconnected existing data tables/files, and the pipeline seems to be running fine. ([view latest two runs](https://app.terra.bio/#workspaces/Nextstrain/Development/job_history)). The new name is more consistent with the `--configfile` flag and documentation, thank you for catching!

Reply:
Nice! If making the change in this PR, just make sure to update other references https://github.com/nextstrain/ncov/pull/929#pullrequestreview-956992360